### ISSUE TITLE
Ensure translations are used in recently added features for 1.0

### ISSUE THREAD

**Issue by Platonn** [SAP/cloud-commerce-spartacus-storefront#2148]
We should ensure that translations mechanism is used in all recently added features, i.r. extensible checkout.

blocked by GH-1809

**Comment by Pucek9** [comment]
I found some disturbing things
![image](https://user-images.githubusercontent.com/8040837/58323279-d12f3d80-7e23-11e9-9e10-a4925ca7d675.png)
![image](https://user-images.githubusercontent.com/8040837/58323507-75b17f80-7e24-11e9-930d-1ffab5ff45a9.png)

**Comment by Pucek9** [comment]
Global message
```json
{
   "errors" : [ {
      "message" : "Cart not found.",
      "reason" : "notFound",
      "subject" : "undefined",
      "subjectType" : "cart",
      "type" : "CartError"
   } ]
}
```

**Comment by kacperknapik** [comment]
@Pucek9 will create a list of problems
@Xymmer to take over

**Comment by Pucek9** [comment]
- [ ] Home breadcumb (most places)
- [ ] Some global messages uses workarounds on frontend (e.g. Cart not found - fixed in #124)
- [ ] Navigation links (Find a store, Sale, Help, Sign Out)
- [ ] Checkout footer links (Need help, Go)
- [ ] Foter links (Frequently asked questions)
- [ ] Copyright (?)

**Comment by Xymmer** [comment]
checking the list - they are all cms except for global messages i think.

**Comment by Xymmer** [comment]
I did some research. To test, when I found the component, I added [DE] in front and change site to German to see if it showed up.

From Michal's list:

Home breadcumb (most places)

Some global messages uses workarounds on frontend (e.g. Cart not found - fixed in #124)

Navigation links (Find a store, Sale, Help, Sign Out)

Checkout footer links (Need help, Go)

- Footer links (Frequently asked questions)
FAQLink component, DE translation is currently English.
![image](https://user-images.githubusercontent.com/4998145/58634081-6022d680-82b8-11e9-8c62-8808c58f665a.png)

- Copyright
